torch/nn/utils/rnn.py

```diff
@@ -91,48 +110,35 @@ def pin_memory(self):
             bind(self.unsorted_indices, lambda t: t.pin_memory()),
         )
 
-    def cuda(self, *args, **kwargs):
-        # Tests to see if 'cuda' should be added to kwargs
-        ex = torch.tensor((), dtype=self.data.dtype, device=self.data.device).to(
-            *args, **kwargs
-        )
-        if ex.is_cuda:
-            return self.to(*args, **kwargs)
-        return self.to(*args, device="cuda", **kwargs)
-
-    def cpu(self, *args, **kwargs):
-        ex = torch.tensor((), dtype=self.data.dtype, device=self.data.device).to(
-            *args, **kwargs
-        )
-        if ex.device.type == "cpu":
-            return self.to(*args, **kwargs)
-        return self.to(*args, device="cpu", **kwargs)
-
-    def double(self):
-        return self.to(dtype=torch.double)
-
-    def float(self):
-        return self.to(dtype=torch.float)
-
-    def half(self):
-        return self.to(dtype=torch.half)
-
-    def long(self):
-        return self.to(dtype=torch.long)
-
-    def int(self):
-        return self.to(dtype=torch.int)
-
-    def short(self):
-        return self.to(dtype=torch.short)
-
-    def char(self):
-        return self.to(dtype=torch.int8)
-
-    def byte(self):
-        return self.to(dtype=torch.uint8)
-
-    def to(self, *args, **kwargs):
```

Comment:
So the story here is that the `int` in the `device` type annotation in one of the `overload` signatures of `PackedSequence.to` was defined after `PackedSequence.int`, and so `mypy` thought that `int` referred to it instead of `builtins.int`...

So I moved `PackedSequence.to` to be defined before `PackedSequence.int`.